src/Kunstmaan/MediaBundle/Entity/Media.php

```diff
@@ -291,6 +291,17 @@ public function getContentType()
     }
 
     /**
+     * @return mixed
+     */
+    public function getType()
+    {
+        $contentType = $this->contentType;
+        $array       = explode("/", $contentType);
+
```

Comment:
This will not work: for example pdf's (content-type: application/pdf) will not be shown when type 'file' is configured.
Before the CMS upgrade the type worked, can you check how this was done in the previous version?